README.md

```diff
@@ -166,4 +197,4 @@ grunt build:dist
 
 This command will not run tests, as it is assumed that running the distribution build process will be triggered after tests have passed, coverage thresholds are maintained and linters and SonarQube checks have been made.
 
-
+ -->
```

Comment:
Arrow?

Reply:
Oh right this is commented out stuff - we should just delete this.

Reply:
yeah that would be better. removed :-)